run.php

```diff
@@ -20,6 +20,9 @@
 const TOOL_URL = 'https://github.com/silverstripe/module-standardiser';
 const PR_TITLE = 'MNT Run module-standardiser';
 const PR_DESCRIPTION = 'This pull-request was created automatically by [module-standardiser](' . TOOL_URL . ')';
+
+// DO NOT change these constants or else new ruleset will be created instead of
```

Comment:
Lol we may as well put them in the JSON files like I said originally then... But no matter, we'll leave as is now.